Cargo.toml

```diff
@@ -32,6 +32,8 @@ derive_builder = "0.20.0"
 getset = "0.1.1"
 strum = "0.26.2"
 strum_macros = "0.26.2"
+regex = "1.10.5"
```

Comment:
I'm worried about performance by introducing regex. WDYT? @saschagrunert

Reply:
Yeah I agree, not sure I like that dependency right here.